index.html

```diff
@@ -6,7 +6,15 @@
     <title>Project Name</title>
   </head>
   <body>
-    <h1>Homepage</h1>
+    <h1>Welcome to Restaurantly</h1>
+    <h2>Delivering great food for more than 18 years!</h2>
```

Comment:
The h1 and h2 should go inside the section tag and the buttons inside section>article
It would look like this:
```suggestion
<section>
     <h1>Welcome to Restaurantly</h1>
     <h2>Delivering great food for more than 18 years!</h2>
     <article>
          <button><a href="./src/pages/menu-specials/"></a>Menu</button>
          <button><a href="./src/pages/events-contacts/"></a>Reserve</button>
     </article>
</section>
```